components/Navbar.vue

```diff
@@ -107,15 +125,68 @@ export default class NavbarMenu extends mixins(PrefixMixin) {
     return this.urlPrefix === 'rmrk' || this.urlPrefix === 'westend'
   }
 
+  get inCollectionPage(): boolean {
+    return this.$route.path.indexOf('/rmrk/collection/') >= 0
+  }
+  get inGalleryDetailPage(): boolean {
+    return (
+      this.$route.path.indexOf('/rmrk/detail') >= 0 ||
+      this.$route.path.indexOf('/rmrk/gallery') >= 0
+    )
+  }
+  get inUserProfilePage(): boolean {
+    return this.$route.path.indexOf('/rmrk/u') >= 0
```

Comment:
instead of using `this.$route.path.indexOf` I would use `this.$route.name` 
- /rmrk/collection/ -> rmrk-collection-id
- /rmrk/detail -> rmrk-gallery-id
- /rmrk/gallery -> rmrk-gallery -id
- /rmrk/u -> rmrk-u-id